.github/workflows/ci.yml

```diff
@@ -507,6 +512,9 @@ jobs:
           - target: i686-unknown-linux-gnu
             host_os: ubuntu-22.04
 
+          - target: riscv64gc-unknown-linux-gnu
+            host_os: ubuntu-22.04
```

Comment:
The previous CI run passed except `coverage`. `coverage` failed because this was missing. I added it and now `coverage` is passing.